lib/daru/view.rb

```diff
@@ -68,21 +68,67 @@ def table_library=(lib)
         end
       end
 
-      # Load the dependent JS files in IRuby notebook. Those JS will help in
-      # plotting the charts in IRuby cell.
+      # dependent script for the library. It must be added in the head tag
+      # of the web application.
       #
       # @example
+      #   dep_js = Daru::View.dependent_script(:highcharts)
+      #   use in Rails app : <%=raw dep_js %>
       #
-      # To load the dependent JS file for Nyaplot library
-      # plotting system (Nyaplot.js, d3.js):
+      # @example
+      #   To load the dependent JS file for Nyaplot library
+      #   plotting system (Nyaplot.js, d3.js):
       #
-      # Daru::View.load_lib_in_iruby('nyaplot')
+      #   Daru::View.dependent_script(:nyaplot)
```

Comment:
Write both examples passing string as well as symbol. In method documentation you should write the input parameter datatypes along with return value details.

Reply:
Done